### PR TITLE
Add neighborhood field to location model

### PR DESCRIPTION
**Pull request by danthaman44**
The new GAM page shows the merchant's neighborhood ("Chinatown", "Sunset Park") in addition to their city/borough. Adding this field
Screen Shot 2020-08-14 at 12 00 16 AM

Test plan:
Screen Shot 2020-08-13 at 11 56 15 PM

ruby/spec/requests/locations_spec.rb passes

### CODE REVIEW

File: db/migrate/20200814034110_add_neighborhood_to_locations.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddNeighborhoodToLocations < ActiveRecord::Migration[6.0]
+  def change
+    add_column :locations, :neighborhood, :string
```
Comment (justintmckibben) [inline db/migrate/20200814034110_add_neighborhood_to_locations.rb:3]: Should we also add borough while we're at it? I know that we sort of sub city for that right now but it's not quite accurate

Comment (justintmckibben) [reply db/migrate/20200814034110_add_neighborhood_to_locations.rb:3]: not a blocker

Comment (justintmckibben) [reply db/migrate/20200814034110_add_neighborhood_to_locations.rb:3]: I guess we can also just use city as borough like we're already doing

Comment (danthaman44) [reply db/migrate/20200814034110_add_neighborhood_to_locations.rb:3]: Yes good call. I did notice that we're using the "City" field to represent borough, which is not scalable lol. Will have a followup PR out for that soon.

**Pull request merged**